### ISSUE TITLE
whether the TDOA2 mode supports remote Settings of the base station

### ISSUE THREAD

**Issue by lalalalang999** [bitcraze/discussions#312]
I can do it witch the tdoa3 model, but the tdoa2 model doesn't work. But I saw the LPP data section in the tdoa2 source code.

**Comment by krichardsson** [comment]
Yes, TDoA2 also supports setting anchor positions from the Crazyflie (same functionality as TDoA3)

**Comment by lalalalang999** [comment]
hello i am a uwb lover. According to the open source code, I transplanted the code of the UWB base station and tag in TDOA released in September 2019, and run it on the dw1000 chip, I did not transplant the code of the latest master branch.
I found that under the TDOA3 algorithm, the tag can realize the remote configuration of the coordinate data of the TDOA3 base station by calling lpsSendLppShort().
However, under the TDOA2 algorithm, I can't remotely configure the TDOA2 base station coordinate data by calling lpsSendLppShort(). What may be the reason why the base station cannot be remotely configured.

Then I compared the processing of LPP packets by TDOA2 and TDOA3 in onEvent(),
By modifying the LPP sending method of the tag under TDAO2, it is modified to detect and send an LPP packet once every onEvent event similar to TDOA3. Call the lpsSendLppShort function more than 100 times within 1s, so that the configuration can sometimes be successful once.
